tools/cbmc/proofs/ARP/ARP_FreeRTOS_OutputARPRequest/OutputARPRequest_harness.c

```diff
@@ -24,7 +52,18 @@ NetworkBufferDescriptor_t xNetworkBuffer;
  * of an ARPPacket to gurantee memory safety.
  */
 NetworkBufferDescriptor_t *pxGetNetworkBufferWithDescriptor( size_t xRequestedSizeBytes, TickType_t xBlockTimeTicks ){
-	xNetworkBuffer.pucEthernetBuffer = malloc(xRequestedSizeBytes);
+	#ifdef CBMC_PROOF_ASSUMPTION_HOLDS
+		#ifdef ipconfigETHERNET_MINIMUM_PACKET_BYTES
+			xNetworkBuffer.pucEthernetBuffer = malloc(ipconfigETHERNET_MINIMUM_PACKET_BYTES);
```

Comment:
Do we want malloc instead of pvPortMalloc?

Reply:
In other ARP proofs and in the TaskPool proofs, we have stubbed out pvPortMalloc to be just malloc.  So in that sense the two are equivalent.  I can do something similar here if reviewers think that is worth doing.